SWLOR.Game.Server/Feature/ChatCommandDefinition/SystemChatCommand.cs

```diff
@@ -180,5 +87,37 @@ private static void ListEmotesCommand(ChatCommandBuilder builder)
                     SendMessageToPC(user, Service.ChatCommand.HelpTextEmote);
                 });
         }
+        private static void StuckCommand(ChatCommandBuilder builder)
+        {
+
+            builder.Create("stuck")
+                .Description("Emergency Escape Command. Use this if you get stuck on a map.")
+                .Permissions(AuthorizationLevel.All)
+                .Validate((user, args) =>
+                {
+                    var lastSubmission = GetLocalString(user, "STUCK_REPORT_LAST_SUBMISSION");                    
+                    if (!string.IsNullOrWhiteSpace(lastSubmission))
+                    {
+                        var dateTime = DateTime.ParseExact(lastSubmission, "yyyy-MM-dd HH:mm:ss", CultureInfo.InvariantCulture);
+
+                        if (DateTime.UtcNow <= dateTime)
+                        {
+                            return "You may only use the stuck command every five minutes. Please wait and try again.";
+                        }                        
+                    }
+
+                    return string.Empty;
+                })
+                .Action((user, target, location, args) =>
+                {
+                    var nextStuckAllowed = DateTime.UtcNow.AddMinutes(5);
```

Comment:
Let's increase this to 30 minutes if you don't mind.

Reply:
Done